README.md

```diff
@@ -4,8 +4,8 @@
 
 Druid is a new Rust-native UI toolkit, still in early stages. Its main
 goal is performance, also aiming for small binary size and compile time,
-fast startup, and very easy build configuration (just `cargo run`). It
-is currently Windows-only, but with hope for porting.
+fast startup, and very easy build configuration (just `cargo run`). 
+It current supports Windows and macOS, with GNU/Linux support planned.
```

Comment:
```suggestion
It currently supports Windows and macOS, with GNU/Linux support planned.
```

Reply:
Thank you
I thought it's weird but decided to not change since I'm not native English speaker.